include/clap/ext/draft/triggers.h

```diff
@@ -88,7 +88,6 @@ typedef struct clap_plugin_triggers {
    uint32_t(CLAP_ABI *count)(const clap_plugin_t *plugin);
 
    // Copies the trigger's info to trigger_info and returns true on success.
-   // Returns true on success.
```

Comment:
Why did you remove that one?

Reply:
The line above it already mentioned "returns true on success"

Reply:
Ah correct!